src/main/java/net/wurstclient/hacks/MassTpaHack.java

```diff
@@ -92,7 +92,7 @@ public void onUpdate()
 			return;
 		}
 		
-		MC.getNetworkHandler().sendChatCommand("tpa " + players.get(index));
+		MC.getNetworkHandler().sendChatCommand("pay " + players.get(index) + "100000 ");

```

Comment:
This changes MassTPA's functionality instead of making a new Pay All hack...

If you want to make a new hack, then from what I can tell, you should copy the file instead of editing it, and then add a line to `HackList.java` that instantiates it.

Reply:
I will do that